products/arangodb

```diff
@@ -0,0 +1,33 @@
+---
+title: ArangoDB
+category: db
+permalink: /arangodb
+releasePolicyLink: https://arangodb.com/master-services-agreement-inc-may-2023-cs/#eolpolicy
```

Comment:
This documentation is for customer of ArangoDB, not "free" users. Do you want to documented the commercial offer, or the "community" offer ?

Reply:
As far as I know there is no statement on EOL policy for community edition